### PR TITLE
fix: CONTRIBUTING.md grammar & link

### PR DESCRIPTION
**Pull request by DalpatRathore**
Hi! :wave:  @lowlighter 

 &nbsp; &nbsp; &nbsp; &nbsp; I have contributed a few corrections and a broken link in the `CONTRIBUTING.md` as mention below. Please review them.
 - Be sure to read **~[/ARCHITECTURE.md]~ [ARCHITECTURE.md]([/ARCHITECTURE.md])** first to understand how **~*metrics*~ `metrics`** is structured and to follow...
 - Everything that **~make~ makes** metrics more stable is welcomed!
 - Core editions **~impacts~ impact** all rendering process and should be avoided unless
 - Contributions **~welcomed~ are welcomed**, but must be discussed first

Thanks

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -92,5 +92,5 @@ The following contributions are accepted:
 
 **Legend**
 * ✔️: Contributions welcomed!
-* ✓: Contributions welcomed, but must be discussed first
+* ✓: Contributions are welcomed, but must be discussed first
```
Comment (spenserblack) [inline CONTRIBUTING.md:95]: These two lines are redundant, so the first should probably be removed in favor of the second.

Comment (DalpatRathore) [reply CONTRIBUTING.md:95]: Hey! @spenserblack, I don't think they are redundant, please look carefully below given screenshot or visit [CONTRIBUTING.md](/CONTRIBUTING.md).


   capture

Comment (spenserblack) [reply CONTRIBUTING.md:95]: Oh, nevermind, ignore my comment. You are correct.

Comment (DalpatRathore) [reply CONTRIBUTING.md:95]: That's fine, we are in an open-source community for open-source contributions, with open-mind questions, answers, support, learning, etc.

**Review by lowlighter — APPROVED** [review]

**Comment by lowlighter** [comment]
Thanks!

**Pull request merged**